docs/reference-guides/block-api/block-registration.md

```diff
@@ -233,11 +233,13 @@ example: {
 
 #### variations (optional)
 
--   **Type:** `Object[]`
+-   **Type:** `Object[]|string`
 -   **Since**: `WordPress 5.9.0`
 
 Similarly to how the block's styles can be declared, a block type can define block variations that the user can pick from. The difference is that, rather than changing only the visual appearance, this field provides a way to apply initial custom attributes and inner blocks at the time when a block is inserted. See the [Block Variations API](/docs/reference-guides/block-api/block-variations.md) for more details.
 
+_Note:_ Starting with WordPress 6.7, it is possible to specify a PHP file that generates the list of block variations on the server side.
```

Comment:
Good point. I think I'll remove the note altogether (and revert the `Type` to just `Object[]`), as its otherwise more confusing. (There's also no documentation about the server-side only `render` field here, so it's probably fine not to mention the server-side-only option for `variations` either.)

Reply:
56f6628b583f6d4e4aebbfb6d1120f559dab8674